### PR TITLE
Add hand generation notbook and an environment file

### PR DESCRIPTION
**Pull request by jhkennedy**
Also, prefer conda for installation in notebooks for the HAND notebooks.

### CODE REVIEW

File: hand-notebook.ipynb [hunk 1]
```diff
@@ -62,28 +62,12 @@
...
-      "https://glo-30-hand.s3.us-west-2.amazonaws.com/v1/2021/Copernicus_DSM_COG_10_N42_00_W087_00_HAND.tif\n",
-      "https://glo-30-hand.s3.us-west-2.amazonaws.com/v1/2021/Copernicus_DSM_COG_10_N42_00_W088_00_HAND.tif\n",
-      "https://glo-30-hand.s3.us-west-2.amazonaws.com/v1/2021/Copernicus_DSM_COG_10_N42_00_W089_00_HAND.tif\n",
-      "https://glo-30-hand.s3.us-west-2.amazonaws.com/v1/2021/Copernicus_DSM_COG_10_N43_00_W087_00_HAND.tif\n",
-      "https://glo-30-hand.s3.us-west-2.amazonaws.com/v1/2021/Copernicus_DSM_COG_10_N43_00_W088_00_HAND.tif\n",
-      "https://glo-30-hand.s3.us-west-2.amazonaws.com/v1/2021/Copernicus_DSM_COG_10_N43_00_W089_00_HAND.tif\n"
-     ]
-    }
-   ],
+   "outputs": [],
```
Comment (asjohnston-asf) [inline hand-notebook.ipynb]: was dropping the output urls intentional? I'd left them in so you get to see example output when browsing the notebook at https://github.com/asjohnston-asf/agu-2022-notebooks/blob/main/hand-notebook.ipynb

File: generate-hand-notebook.ipynb [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## 1. Generate HAND\n",
+    "First select a DEM tile and then provide the [virtual path](https://gdal.org/user/virtual_file_systems.html) to it:"
```
Comment (jacquelynsmale) [inline generate-hand-notebook.ipynb]:
```suggestion
    "First, select a DEM tile and then provide the [virtual path](https://gdal.org/user/virtual_file_systems.html) to it"
```

File: generate-hand-notebook.ipynb [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+    "out_pixels = np.ma.masked_where(wbm_pixels == 1, out_pixels)"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Write out our final HAND tile"
```
Comment (jacquelynsmale) [inline generate-hand-notebook.ipynb]:
```suggestion
    "Write out the final HAND tile"
```

File: hand-notebook.ipynb (hunk 1 above)
Comment (jhkennedy) [reply hand-notebook.ipynb]: Nah, I accidentally blew them away. I'll put them back; just need to edit it by hand (or get the STAC catalog up and running and run it!)

**Pull request merged**